connectors/opa/opa_connector.go

```diff
@@ -239,20 +228,36 @@ func buildNewEnfrocementAction(transformAction interface{}) (*pb.EnforcementActi
 			log.Println("Warning: unknown format of used policy information. Skipping policy", action)
 		}
 
-		if result, ok := action["result"].(string); ok {
+		if result, ok := action["action_name"].(string); ok {
 			switch result {
-			case "Remove column":
-				if columnName, ok := extractArgument(action["args"], "column name"); ok {
+			case "remove column":
```

Comment:
There are too many hard-coded strings ("remove column", "removed-ID", "removed", ...). I suggest to have them in a separate file, as enums.

Reply:
My personal suggestion: This PR deals with  conceptual OPA changes that were done by Ety and are pending since a long time. We can focus on conceptual level comments in this PR I think and if there are no conceptual level comments, the changes in this PR can to be accepted as quickly as possible so that newer policies are designed according to this new framework. 
I think we can take the above comment in another PR which is basically related to enhance the new design of OPA. 
However, if there is a need to handle these comments in this PR itself, please let me know and I can make these changes in this PR itself.

Reply:
OK.

Reply:
@rohithdv I believe this is related to the comments that I gave in the original PR.  We need a taxonomy for the governance actions instead of having them hardcoded.

Reply:
Can't it all be copied from action as-is without special casing? (Name to name, argument to args, etc.)